src/Components/ArticleComments.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react'
+import { fetchArticleComments } from '../api'
+import { useParams } from 'react-router-dom'
+import Comment from './Comment'
+
+
+function ArticleComments() {
+    const [comments, setComments] = useState([])
+    const {article_id} = useParams()
+    useEffect(() => {
+        fetchArticleComments(article_id).then(({ comments }) => {
```

Comment:
Need some loading/error state to communicate to a user when the request is in progress or if the request has failed